Content.Server/GameTicking/Commands/SetGamePresetCommand.cs

```diff
@@ -8,20 +8,27 @@ namespace Content.Server.GameTicking.Commands
     sealed class SetGamePresetCommand : IConsoleCommand
     {
         public string Command => "setgamepreset";
-        public string Description => "";
-        public string Help => "";
+        public string Description => Loc.GetString("set-game-preset-command-description", ("command", Command));
+        public string Help => Loc.GetString("set-game-preset-command-help-text", ("command", Command));
 
         public void Execute(IConsoleShell shell, string argStr, string[] args)
         {
             if (args.Length != 1)
             {
-                shell.WriteLine("Need exactly one argument.");
+                shell.WriteError(Loc.GetString("shell-wrong-arguments-number"));
```

Comment:
Add a new localization that takes the exact number of arguments needed so it's not so vague.